controllers/noobaa-rules.yaml

```diff
@@ -0,0 +1,211 @@
+         annotations:
+           description: A NooBaa bucket {{ $labels.bucket_name }} is using {{ printf
+             "%0.0f" $value }}% of its capacity
+           message: A NooBaa Bucket Is In Low Capacity State
+           severity_level: warning
+           storage_type: NooBaa
+         expr: |
+           NooBaa_bucket_capacity{bucket_name=~".*"} > 80
+         for: 5m
+         labels:
+           severity: warning
+       - alert: NooBaaBucketNoCapacityState
+         annotations:
+           description: A NooBaa bucket {{ $labels.bucket_name }} is using all of its
+             capacity
+           message: A NooBaa Bucket Is In No Capacity State
+           severity_level: warning
+           storage_type: NooBaa
+         expr: |
+           NooBaa_bucket_capacity{bucket_name=~".*"} > 95
+         for: 5m
+         labels:
+           severity: warning
+   - name: resource-state-alert.rules
+     rules:
+       - alert: NooBaaResourceErrorState
+         annotations:
+           description: A NooBaa resource {{ $labels.resource_name }} is in error state
+             for more than 5m
+           message: A NooBaa Resource Is In Error State
+           severity_level: warning
+           storage_type: NooBaa
+         expr: |
+           NooBaa_resource_status{resource_name=~".*"} == 0
+         for: 5m
+         labels:
+           severity: warning
+       - alert: NooBaaNamespaceResourceErrorState
+         annotations:
+           description: A NooBaa namespace resource {{ $labels.namespace_resource_name
+             }} is in error state for more than 5m
+           message: A NooBaa Namespace Resource Is In Error State
+           severity_level: warning
+           storage_type: NooBaa
+         expr: |
+           NooBaa_namespace_resource_status{namespace_resource_name=~".*"} == 0
+         for: 5m
+         labels:
+           severity: warning
+   - name: system-capacity-alert.rules
+     rules:
+       - alert: NooBaaSystemCapacityWarning85
+         annotations:
+           description: A NooBaa system is approaching its capacity, usage is more than
+             85%
+           message: A NooBaa System Is Approaching Its Capacity
+           severity_level: warning
+           storage_type: NooBaa
+         expr: |
+           NooBaa_system_capacity > 85
+         for: 5m
+         labels:
+           severity: warning
+       - alert: NooBaaSystemCapacityWarning95
+         annotations:
+           description: A NooBaa system is approaching its capacity, usage is more than
+             95%
+           message: A NooBaa System Is Approaching Its Capacity
+           severity_level: critical
+           storage_type: NooBaa
+         expr: |
+           NooBaa_system_capacity > 95
+         for: 5m
+         labels:
+           severity: critical
+       - alert: NooBaaSystemCapacityWarning100
+         annotations:
+           description: A NooBaa system approached its capacity, usage is at 100%
```

Comment:
Maybe we do not need this alert..Are we sure that the capacity can be used completely, I mean, probably we are going to have always some bytes "free". In any case it does not hurt ... this is a simple comment.